integration-tests/sceptre-project/config/external-stack-output/resolver-no-profile-region.yaml

```diff
@@ -0,0 +1,5 @@
+template:
```

Comment:
This was the new testcase we needed, using !stack_output_external with no explicit profile/region/role.